build/search-provider.js

```diff
@@ -0,0 +1,34 @@
+'use strict';
+
+/**
+ *  This module generates 'search_providers.json' used by the Search and
+ *  Settings Applications
+ *
+ */
+
+/* global require, exports */
+const utils = require('utils');
+
+const IN_FILE = 'search_providers_input.json';
+const OUT_FILE = 'search_providers.json';
+
+function generateSearchProviderJson(opts) {
+
+  var variantFile = utils.getFile(opts.GAIA_DISTRIBUTION_DIR, 'variant.json');
```

Comment:
Seems that the build tests are not happy about this file, build/utils.js: Invalid file path (, variant.json).